lib/common/util.js

```diff
@@ -119,14 +119,20 @@ exports.timestamp = function (aDate) {
  * @param  {Publisher} publisher    Publisher module via changefeed
  * @param  {string}    resource     Identifies the resource e.g. 'vms'
  * @param  {array}     subResources Strings of resource properties e.g ['nics']
+ * @param  {object}    resObj       The vm which just changed
```

Comment:
nit: this line should be added after the uuid line (i.e. same ordering as the function)

Reply:
Done!